src/main/kotlin/com/frcteam3636/frc2025/subsystems/drivetrain/Drivetrain.kt

```diff
@@ -354,63 +357,53 @@ object Drivetrain : Subsystem, Sendable {
 
     private val alignController = PIDController(Constants.ALIGN_PID_GAINS)
 
-    fun alignToClosestPOI(sideOverride: ReefBranchSide? = null): Command = defer {
-        val target = AprilTagTarget.currentAllianceTargets
-            .asIterable()
-            .closestTargetToWithSelection(estimatedPose, sideOverride ?: currentTargetSelection)
-
-        Logger.recordOutput("Drivetrain/Auto-align Target", target.pose)
-        AutoBuilder.pathfindToPose(target.pose, DEFAULT_PATHING_CONSTRAINTS, 1.5.metersPerSecond)
-            .andThen(runOnce {
-                alignController.reset()
-                Logger.recordOutput("/Drivetrain/Align-Running", true)
-            })
-            .andThen(runEnd({
-                val distanceToTarget = estimatedPose.relativeTo(target.pose).translation.norm
-                val angleToTarget = estimatedPose.relativeTo(target.pose).translation.angle
-                val output = alignController.calculate(distanceToTarget, 0.0)
-                val desiredSpeed = Translation2d(output, angleToTarget)
-                desiredChassisSpeeds = ChassisSpeeds(
-                    desiredSpeed.x,
-                    desiredSpeed.y,
-                    0.0
-                )
-            }, {
-                desiredModuleStates = BRAKE_POSITION
-                Logger.recordOutput("/Drivetrain/Align-Running", false)
-            }))
-//            .onlyWhile {
-//                estimatedPose.relativeTo(target.pose).translation.norm >
-//                        Inches.of(0.1).meters
-//            }
+    fun alignToClosestPOI(sideOverride: ReefBranchSide? = null, usePathfinding: Boolean = true) =
+        alignToTarget(usePathfinding) {
+            AprilTagTarget.currentAllianceTargets.asIterable()
+                .closestTargetToPoseWithSelection(estimatedPose, sideOverride ?: currentTargetSelection).pose
+        }
+
+    fun alignToBarge() = alignToTarget {
+        BargeTarget.closestTo(
+            estimatedPose.translation,
+            DriverStation.getAlliance().getOrDefault(DriverStation.Alliance.Blue)
+        ).pose
     }
 
-    fun alignToTargetWithPIDController(sideOverride: ReefBranchSide? = null): Command = defer {
-        val target = AprilTagTarget.currentAllianceTargets
-            .asIterable()
-            .closestTargetToWithSelection(estimatedPose, sideOverride ?: currentTargetSelection)
+    fun alignToTarget(usePathfinding: Boolean = true, target: () -> Pose2d): Command = defer {
```

Comment:
Do we really want to introduce breaking changes? I don't feel comfortable merging this as we have STEMnasium tomorrow and we need all the testing time we can get. Debugging this would be a waste.

Reply:
To clarify I mean we should probably not be restructuring all the auto alignment stuff right now.

Reply:
If you'd like you can read over the changes and look for mistakes. If you don't see any, we can test if it works tomorrow.